client/extensions/woocommerce/lib/order-status/index.js

```diff
@@ -68,7 +68,17 @@ export function getOrderStatusList() {
  * @return {Boolean} true if the status is awaiting payment
  */
 export function isOrderWaitingPayment( status ) {
-	return ( -1 !== statusWaitingPayment.indexOf( status ) );
+	return -1 !== statusWaitingPayment.indexOf( status );
```

Comment:
Curious - why not test for string equality?

Reply:
What do you mean?